lib/inventory_refresh/save_collection/saver/sql_helper.rb

```diff
@@ -142,6 +142,12 @@ def all_entities_query(all_manager_uuids_scope)
           all_entities_query = all_entities_query.where(condition)
         end
 
+        if all_manager_uuids_timestamp && supports_column?(:resource_timestamp)
+          all_manager_uuids_timestamp = Time.parse.utc(all_manager_uuids_timestamp)
```

Comment:
Should be Time.parse(all_manager_uuids_timestamp).utc

Reply:
yea :-D